### PR TITLE
jython: add httpsender template script

### PR DESCRIPTION
**Pull request by thc202**
Add a Python HTTP Sender template script (same behaviour as JavaScript
template).
Update changes and files in ZapAddOn.xml file.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/jython/ZapAddOn.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<description>Allows Python to be used for ZAP scripting - templates included</description>
 	<author>ZAP Dev Team</author>
 	<url></url>
-	<changes>Minor code changes</changes>
+	<changes>Add HTTP Sender template script.</changes>
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/jython/ZapAddOn.xml]: I believe either the removed line should be left or the version should be updated.

File: src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# The sendingRequest and responseReceived functions will be called for all
+# requests/responses sent/received by ZAP, # including automated tools (e.g.
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py]: Remove hash tag and space before "including?"

File: src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#      7   CHECK_FOR_UPDATES_INITIATOR
+#      8   BEAN_SHELL_INITIATOR
+#      9   ACCESS_CONTROL_SCANNER_INITIATOR
+# For the latest list of values see the HttpSender class:
+# https://github.com/zaproxy/zaproxy/blob/master/src/org/parosproxy/paros/network/HttpSender.java
+# 'helper' just has one method at the moment: helper.getHttpSender() which
+# returns the HttpSender instance used to send the request.
+#
+# New requests can be made like this:
+# msg2 = msg.cloneAll() # msg2 can then be safely changed without affecting msg
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py]: Second hash tag on seperate line?

File: src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+# helper.getHttpSender().sendAndReceive(msg2, false)
+# print('msg2 response=' + msg2.getResponseHeader().getStatusCode())
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py]: `'msg2 response code='` ?

File: src/org/zaproxy/zap/extension/jython/ZapAddOn.xml (hunk 1 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/jython/ZapAddOn.xml]: Removed the previous change as that's no longer needed IMHO, the previous change was just added to not leave the changes empty after bumping the version (I don't think that needs to be noted in the changes other than in this case). If you think it's better leave it I'll add it again.

File: src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py (hunk 2 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py]: Changed.

File: src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py (hunk 3 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py]: That's on purpose, it's to be a comment for the statement (but can move it anyway).

File: src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py (hunk 4 above)
Comment (thc202) [inline src/org/zaproxy/zap/extension/jython/files/scripts/templates/httpsender/HttpSender default template.py]: Changed.

File: src/org/zaproxy/zap/extension/jython/ZapAddOn.xml (hunk 1 above)
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/jython/ZapAddOn.xml]: K

**Pull request merged**